draft-ietf-lamps-x509-shbs.md

```diff
@@ -212,11 +220,57 @@ Note that the `id-alg-hss-lms-hashsig` algorithm identifier is also referred to
 as `id-alg-mts-hashsig`. This synonym is based on the terminology used in an
 early draft of the document that became [RFC8554].
 
+The public key and signature values identify the hash function used in the
```

Comment:
Thank you for putting this sentence in. This will help the reader. I would even add that the tree height of the S-HBS instance is also determined in the public key and the signature. That is, we write: 

The public key and signature values identify the hash function and the height used in the HSS/LMS tree.